recipes/pydm/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "pydm" %}
+{% set version = "1.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slaclab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 0274ad8f906116bc03245478585ad596c7b89994a99ea3cd73f990efef6ecaf6
```

Comment:
Can this be `noarch: python`?

Reply:
I think so... Would this be a problem for `pyqt`?
I'm happy to change it...

Reply:
I don't think so.  If you are not compiling anything in this recipe, then it can be noarch python, generally speaking.

Reply:
Awesome... I changed it to `noarch: python`